### PR TITLE
[DOC] Add FAQ about working offline locally

### LINKED ISSUE #5510: [BUG] Running localhost, cannot make request when no internet connection
### Describe the bug

I running a refine app on localhost, it should run normal even there's no internet connection

But i cant use the app without internet, i see there is telemetry req error while no connection theni turn off the telemetry through (`Refine` -> `options` -> `disableTelemery: true`). but still not solving the problem, when i see in network console, the requset not start until i connect to internet

### Steps To Reproduce

1. Install refine
2. Setup data and auth provider
3. Login `Success!`
5. Logout
6. Turn off internet connection
7. Retry Login
8. Just loading `Not Working!`
9. Check the network console, no request whatsoever is initiated
10. Retry login
11. Turn on internet connection
12. Check the nework console, request is started
13. Login `Success`

### Expected behavior

It should run normal even in no connection

### Screenshot

_No response_

### Desktop

OS: Linux Ubuntu
Browser: Chrome Latest

### Mobile

_No response_

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by ilomon10**
Add a common question around using library
closes #5510

### CODE REVIEW

File: documentation/docs/guides-concepts/faq/index.md [hunk 1]
```diff
@@ -369,7 +369,37 @@ When working with forms and mutations, using only client side validation might n
 
 To learn more about server side validation and see an example, check out the [Server Side Validation section of the Forms guide.](/docs/guides-concepts/forms/#server-side-validation-)
 
+## How to work with Local environtment?
```
Comment (BatuhanW) [inline documentation/docs/guides-concepts/faq/index.md:372]:
```suggestion
## How to work offline in local environment?
```

File: documentation/docs/guides-concepts/faq/index.md [hunk 2]
```diff
@@ -369,7 +369,37 @@ When working with forms and mutations, using only client side validation might n
...
+
+When working without a network connection, `@tanstack/query` prevents requests from being made until a successful connection is established. You might feel frustrated and wonder,
+
+> WHY!!! can't I work with my own data? Must I use an internet connection to run this library?
```
Comment (BatuhanW) [inline documentation/docs/guides-concepts/faq/index.md:376]:
```suggestion
> can't I work with my own data? Must I use an internet connection to run this library?
```

😄 I really appreciate the candid tone in this section, but let's consider removing the `WHY!!!` part to maintain neutral tone in the documentation.

**Review by BatuhanW — COMMENTED** [review]
Hey @ilomon10  Thanks for the PR! Added a minor comments, then we'll merge it.

File: documentation/docs/guides-concepts/faq/index.md (hunk 2 above)
Comment (ilomon10) [reply documentation/docs/guides-concepts/faq/index.md:376]: i think its better for search engine indexing 😅

Comment (ilomon10) [reply documentation/docs/guides-concepts/faq/index.md:376]: maybe its better just use leave the word like 
> Why can't  I ....

Comment (BatuhanW) [reply documentation/docs/guides-concepts/faq/index.md:376]: Yeah, that looks better!

**Review by BatuhanW — APPROVED** [review]

**Comment by BatuhanW** [comment]
Thanks for the contribution, @ilomon10!

**Pull request merged**